builtin/settingtypes.txt

```diff
@@ -490,38 +648,35 @@ max_packets_per_iteration (Max. packets per iteration) int 1024
 
 [*Game]
 
-#    Default game (default when creating a new world).
+#    Default game when creating a new world.
+#    This will be overridden when creating a world from the main menu.
 default_game (Default game) string minetest
 
-#    Message of the Day.
-motd (Message of the day) string Welcome to this awesome Minetest server!
+#    Message of the day displayed to players connecting.
+motd (Message of the day) string
 
-#    Maximum number of players connected simultaneously.
+#    Maximum number of players that can connect simultaneously.
 max_users (Maximum users) int 15
```

Comment:
I'd argue this is a server setting, rather than a game setting: same for motd.

"game" for me means settings related to gameplay.